### PR TITLE
Htmlcss ex

### PR DESCRIPTION
**Pull request by julianpoma**
Hi Maxi.

I created this new branch in order to get all the excersises organized.

I fixed all your previous corrections on the HTML excersises, and i added the CSS ones. 

See you tomorrow!

### CODE REVIEW

File: 00-HTML-CSS/Ex-2.5.2/styles.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+.clearfix {
+  *zoom: 1;
+}
+.images {
+  width: 100%;
+  height: 100%;
+}
+.screen {
+
```
Comment (maximilianotulian) [inline 00-HTML-CSS/Ex-2.5.2/styles.css]: Minor thing, remove this blank line

File: 00-HTML-CSS/Ex-2.5.2/styles.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    width: 100%;
+    position: absolute;
+    opacity: 0;
+    z-index: 1000;
+    -webkit-transition: all 0.3s ease-in-out;
+    -moz-transition: all 0.3s ease-in-out;
+    -o-transition: all 0.3s ease-in-out;
+    -ms-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    width: 100%;
```
Comment (maximilianotulian) [inline 00-HTML-CSS/Ex-2.5.2/styles.css]: Please remove the style duplicated

File: 00-HTML-CSS/Ex-2.5.2/styles.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  content: " ";
+  display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+.clearfix {
+  *zoom: 1;
```
Comment (maximilianotulian) [inline 00-HTML-CSS/Ex-2.5.2/styles.css]: Interesting, `*`

File: 00-HTML-CSS/Ex-2.5.2/index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+<head>
+    <link href="normalize.css" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8">
+    <title>Exercise 2.5.2</title>
+</head>
+<body>
+    <div class="container">
+        <div class="row clearfix">
```
Comment (maximilianotulian) [inline 00-HTML-CSS/Ex-2.5.2/index.html]: We missed the definition of `row` in the css

**Comment by julianpoma** [comment]
hi maxi

"The next time try to not use style attributes inside the html elements."
Fixed

"The for attribute should point to an element id"
Ok. I though it was the label identifier, that lets you access and modify its value. Like on C#.

"This is a warning. The normalize should be required first. Anyway you did it correctly in the following exercise."
Fixed and thanks!

"The last two selectors have an addition tab. Try to keep consistency across the files."
Noted!

"There is not need to use !important. Remember to keep the specificity as lower as possible."
mistype there. I was trying things.

"Why we're using double qoutes? If you want to highlight the sentence is right, the other way we should remove it."
Another mistype. Fast copy and paste!

"Question, what happen if I change the float from right to left?"
It will move to the left and be next to the previous box. It will go down if it hasnt got any space left.

"Whenever we use an img tag, we should add an alternative description"
Roger !

"Copy and paste is really useful, just be aware of its side effects. Please check the class of the second <p>"
Fixed! Missed " "

"Please remove the style duplicated"
Removed

"We missed the definition of row in the css"
Removed!

File: 00-HTML-CSS/Ex-1/signup.html [hunk 5]
```diff
@@ -8,15 +8,15 @@
...
+      <label for="">Last name:</label><br>
       <input type="text" name="lname" value="" required><br><br>
-      <label for="email">Valid email:</label><br>
+      <label for="">Valid email:</label><br>
       <input type="email" name="email" value="" placeholder="something@domain.whatever" required><br><br>
-      <label for="birth">Birthday:</label><br>
+      <label for="">Birthday:</label><br>
       <input type="date" name="birth" placeholder="dd/mm/aaaa"><br><br> <!-- doesnt work on firefox, thats why the placeholder -->
-      <label for="sport">Favourite sport:</label><br>
+      <label for="">Favourite sport:</label><br>
```
Comment (maximilianotulian) [inline 00-HTML-CSS/Ex-1/signup.html]: Maybe I make myself unclear, the label should point to tag id. For example, the first label you have is for "fname". The input first name should have the id="fname". Anyway is something minor, not needed to change it. Remember it for the next ocassion.

Comment (julianpoma) [inline 00-HTML-CSS/Ex-1/signup.html]: Ok maxi, thank you and got it. Can we merge this branch with master?

**Pull request merged**